### PR TITLE
Cordova Plugin Update

### PR DESCRIPTION
**Pull request by ndesai-newrelic**

### CODE REVIEW

**Review by JustinRush-NR — COMMENTED** [review]
LGTM

File: plugin.xml [hunk 1]
```diff
@@ -1,41 +1,53 @@
...
     <framework src="CoreTelephony.framework" />
     <framework src="SystemConfiguration.framework" />
     <framework src="libz.dylib" />
     <framework src="libc++.dylib" />
+    <podspec>
+      <config>
+        <source url="https://cdn.cocoapods.org/" />
+      </config>
+      <pods use-frameworks="true">
+        <pod name="NewRelicAgent" spec="7.3.8" />
```
Comment (cdillard-NewRelic) [inline plugin.xml:48]: 7.4.0 is the latest version.

File: src/ios/NewRelicCordovaPlugin.m [hunk 2]
```diff
@@ -17,15 +17,224 @@ - (void)pluginInitialize
...
     if (applicationToken == nil || ([applicationToken isEqualToString:@""] || [applicationToken isEqualToString:@"x"])) {
-    	NRLOG_ERROR(@"Failed to load application token! The iOS agent is not configured for Cordova.");
+        NRLOG_ERROR(@"Failed to load application token! The iOS agent is not configured for Cordova.");
+        
+    } else {
+        [NewRelic setPlatform:NRMAPlatform_Cordova];
+        [NewRelic setPlatformVersion:platformVersion];
+        [NewRelic startWithApplicationToken:applicationToken];
+    }
+}
```
Comment (cdillard-NewRelic) [inline src/ios/NewRelicCordovaPlugin.m:29]:
```suggestion
}

```

**Review by cdillard-NewRelic — APPROVED** [review]

**Review by cthomas-newrelic — APPROVED** [review]
🐑 it

**Pull request merged**